### PR TITLE
added Italian and German months

### PR DESCRIPTION
**Pull request by retokromer**

### CODE REVIEW

File: lib/timetwister/utilities.rb [hunk 1]
```diff
@@ -131,6 +131,34 @@ def self.language_to_english(str)
 			'octubre' => 'October',
 			'noviembre' => 'November',
 			'diciembre' => 'December'
```
Comment (ablwr) [inline lib/timetwister/utilities.rb]: comma?

File: lib/timetwister/utilities.rb [hunk 2]
```diff
@@ -131,6 +131,34 @@ def self.language_to_english(str)
...
+			'marzo' => 'March',
+			'aprile' => 'April',
+			'maggio' => 'May',
+			'giugno' => 'June',
+			'luglio' => 'July',
+			'agosto' => 'August',
+			'settembre' => 'September',
+			'ottobre' => 'October',
+			'novembre' => 'November',
+			'dicembre' => 'December'
```
Comment (ablwr) [inline lib/timetwister/utilities.rb]: and comma!

**Comment by alexduryee** [comment]
Specs were failing because the German months of `januar` and `februar` were matching on `january` and `february`.  I fixed the regex so the German months no longer match on English months.

Many thanks for the PR!

**Comment by retokromer** [comment]
Thank you for the information, @alexduryee! I was wondering...

**Pull request merged**